dags/sentinel-2_indexing.py

```diff
@@ -69,6 +69,17 @@
     """)
 ]
 
+ARCHIVE_BASH_COMMAND = [
+    "bash",
+    "-c",
+    dedent("""
+        datacube dataset search -f csv 'product=s2_l2a lon in [170,200]' > /tmp/to_kill.csv;
```

Comment:
Shouldn't this be [170, 190]?

Reply:
Could be. Doesn't really matter, as there shouldn't be anything higher than ~181 anyway!